pom.client.xml

```diff
@@ -684,31 +712,69 @@
       </reporting>
     </profile>
 
-    <!-- Setup for Java 9+ -->
+    <!-- Setup for Java 11+ -->
     <profile>
       <id>java-lts</id>
       <activation>
-        <jdk>[9,)</jdk>
+        <jdk>[11,)</jdk>
```

Comment:
So, we will not support building on JDK 9 or 10? The profiles are now only for 8 and 11+.

Reply:
I will need to test again on 9. Previously I ran into bugs in 9 that prevented compilation.